ui/app/AppLayouts/Wallet/views/LeftTabView.qml

```diff
@@ -333,37 +335,50 @@ Rectangle {
                         radius: Style.current.radius
                         color: header.highlighted || mouseArea.containsMouse ? Style.current.backgroundHover : root.color
                         implicitWidth: parent.ListView.view.width - Style.current.padding * 2
-                        implicitHeight: parent.ListView.view.firstItem.height + Style.current.padding
                     }
 
-                    contentItem: Item {
+                    contentItem: ColumnLayout {
+                        spacing: -1
```

Comment:
Hmm?

Reply:
as per design https://www.figma.com/file/FkFClTCYKf83RJWoifWgoX/Wallet-v2?type=design&node-id=12970-505211&mode=dev :D

Reply:
You most likely want just `0`; `-1` is sort of invalid, even though Figma might use that

Reply:
<img width="392" alt="image" src="https://github.com/status-im/status-desktop/assets/60327365/7f5ac419-aa02-4d61-8fd1-9bf9bf7f095d">

Ive adjusted other values to acheive the same effect now.